pagefaultgames/pokerogue
# [Localization] Fix for English 'Battles Won', updated tutorial # of Pokerus mons from 3 -> 5

## What are the changes the user will see?
Fixed Battles Won # in the text for english
Fixes # of Pokerus total from 3 -> 5

## Why am I making these changes?
Needed to be fixed before beta merge, localization fix.

## What are the changes from a developer perspective?
Nothing

## How to test the changes?
Merge to local, reload main menu until you get '# Battles Won', then check with tutorial on and try to select a mon with pokerus.

## Checklist
